packages/checkout-ui-extensions/src/components/View/View.ts

```diff
@@ -149,6 +254,41 @@ export interface ViewProps extends Bordered {
    * - `1` represents `100%`
    */
   blockSize?: number | Responsive<number>;
+
+  /**
+   * Sets the opacity of the View. The opacity will be applied to the background as well as all
+   * the childrens of the View. Use carefully as this could decrease the contrast ratio between
+   * the background and foreground elements resulting in unreadable and inaccessible text.
+   */
+  opacity?: Opacity;
+  /**
+   * A label that describes the purpose or contents of the element. When provided,
+   * it will be announced to buyers using assistive technologies and will
+   * provide them with more context.
+   */
+  accessibilityLabel?: string;
+  /**
+   * Set the semantic of the component’s content
+   */
+  accessibilityRole?:
```

Comment:
@jffortier do you know what the tuple version of this type means?

Reply:
Yes!

- when one value is set, we render the correct HTML element if one exist for that specific role. ex: `<View accessibilityRole="header">` will render `<header>`
- when a second value is set, we then set the `role` property on the element. ex: `<View accessibilityRole={['header', 'banner']}>` will render `<header role="banner">`

It can be useful for many use cases.  In Self Serve, they need the list/listItem semantic _and_ the stylistic flexibility of View for their UI. They also needed a list item that had the role of "separator".

```tsx
<View accessibilityRole="list" {...styleOnlyPossibleWithView}>
   <View accessibilityRole="listItem">...</View>
   <View accessibilityRole={{'listItem', 'separator']} border="base" />
   <View accessibilityRole="listItem">...</View>
</View>
```

Which renders
```html
<ul>
   <li>...</li>
   <li role="separator" class="css-class-for-border" />
   <li>...</li>
</ul>
```

Reply:
Hmm, interesting. I think we should probably add some of this content as documentation for the prop, unless a deeper explainer is somewhere else. It wasn't clear to me what they meant, and I still find it a bit hard to understand from this type what values are actually useful to use in the two-value format (e.g., `['listItem', 'listItem']` is valid here but presumably not a thing you'd ever want in practice?)

Reply:
> I think we should probably add some of this content as documentation for the prop, unless a deeper explainer is somewhere else. 

Yeah, I remember providing that feedback in the PR, but now can't find the detailed explanation in the code. Not sure what happened. I'll add one.

>  I still find it a bit hard to understand from this type what values are actually useful to use in the two-value format (e.g., ['listItem', 'listItem'] is valid here but presumably not a thing you'd ever want in practice?)

We can provide examples in that description. Yes, `['listItem', 'listItem']` is valid with this API. We currently allow it (eg:   this will render `<li role="listItem">`, but could decide to not do anything with the second value when its identical to the first.